docs/dictionary/command/androidRequestPermission.lcdoc

```diff
@@ -0,0 +1,45 @@
+Name: androidRequestPermission
+
+Type: command
+
+Syntax: androidRequestPermission(<permissionName>)
+
+Summary:
+Displays a dialog showing a permission request for <permissionName>. If a user has already granted permission for <permissionName>, this command does nothing.
+
+Introduced: 9.0.1
+
+OS: android
+
+Platforms: mobile
+
+Example:
+local tCameraPermissionGranted
+put androidHasPermission("android.permission.CAMERA") into tCameraPermissionGranted
+if not tCameraPermissionGranted then
+   androidRequestPermission("android.permission.CAMERA")
+end if
+put androidHasPermission("android.permission.CAMERA") into tCameraPermissionGranted
+if not tCameraPermissionGranted then
+   answer "This app is not permitted to access the device camera. You can change this" && \
+         "in the Settings app."
+end if
+
+
+Parameters:
+permissionName (enum):
+The name of the permission to request.
+
+-   "android.permission.CAMERA": permission to access the device camera.
```

Comment:
@livecodepanos should this list all the dangerous permissions: https://developer.android.com/guide/topics/permissions/overview#permission-groups